apps/api/src/main.ts

```diff
@@ -1,8 +1,20 @@
 import { NestFactory } from '@nestjs/core';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
```

Comment:
we're not actually running validation on input without this haha
```suggestion
  app.useGlobalPipes(new ValidationPipe());
```

Reply:
should be fixed